lib/provider.js

```diff
@@ -182,7 +182,9 @@ const init = (providerConfig) => {
             typeof config.metadata === 'function'
               ? config.metadata(file)
               : {
-                  contentDisposition: `inline; filename="${file.name}"`,
+                  contentDisposition: `inline; filename="${file.name
+                    .normalize('NFD')
+                    .replace(/[\u0300-\u036f]/g, '')}"`,
```

Comment:
I think is better to create a variable named with what this regex really does.

has better readability for when you need maintenance

Reply:
You're right! Thanks :)